File: TPTD3/exo/Entrepot.java
public class Entrepot{
    private int code;
    private String nom;
    private String departement;
    public Entrepot(int c,String n,String d){
        this.code = c;
        this.nom = n;
        this.departement = d;
    }
    public int getCode(){
        return this.code;
    }
    public void setCode(int c){
        this.code = c;
    }
    public String getNom(){
        return this.nom;
    }
    public void setNom(String n){
        this.nom = n;
    }
    public String getDepartement() {
        return departement;
    }
    public void setDepartement(String departement) {
        this.departement = departement;
    }
    @Override
    public String toString() {
        return "Entrepot [code=" + code + ", nom=" + nom + ", departement=" + departement + "]";
    }
}

File: TPTD4/Entrepot.py
class Entrepot:
    def __init__(self,code,nom,dep) -> None:
        self._code = code
        self._nom = nom
        self._dep = dep
    
    def __repr__(self) -> str:
        return "Entrepot %d nommé %s qui vient du departement %s"+"\n"

    @property
    def getCode(self):
        return self._code
    
    @property
    def getNom(self):
        return self._nom
    
    @property
    def getDep(self):
        return self._dep
    
    def setCode(self,c):
        self._code = c
    
    def setNom(self,n):
        self._nom = n
    
    def setDep(self,d):
        self._dep = d